lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -5,6 +5,7 @@
 from openedx.core.djangolib.markup import HTML, Text
 from common.djangoapps.course_modes.models import CourseMode
 from common.djangoapps.util.course import should_display_grade
+from lms.djangoapps.certificates.data import CertificateStatuses
```

Comment:
Are further changes missing here? Did we want to compare the statuses on the lines below (line 25, 27, 29, etc.) using the CertificateStatuses enum? Or was this captured in another PR?

Reply:
This PR is just the refactor of the code. I am guessing this remained while I was cherry-picking the changes from my other PR. I will remove this from this PR.